### PR TITLE
[WIP] Update extract-text-webpack-plugin to v3, drop webpack 2 support

### LINKED ISSUE #190: Update extract-text-webpack-plugin
When using webpack 3 I have a warning:

```
npm WARN extract-text-webpack-plugin@2.1.2 requires a peer of webpack@^2.2.0 but none was installed.
```

The problem is the new extract-text-webpack-plugin has [some breaking changes](https://github.com/webpack-contrib/extract-text-webpack-plugin/releases/tag/v3.0.0) and requires webpack 3.1. Which means if we update it webpack-blocks packages would require webpack 3.1 too.

### PR DESCRIPTION
**Pull request by sapegin**
We need to make sure we have a version of `extractText` block that *still* works in webpack 2, which should be published before merging this PR, if have any unpublished changes.

Probably I should update webpack peer dependency in all other blocks.

Probably we should update the readme to say which versions of the block work in which versions of webpack.

Closes #190.

### CODE REVIEW

**Comment by andywer** [comment]
> Probably we should update the readme to say which versions of the block work in which versions of webpack.

Yes, we should do that! :)

> Probably I should update webpack peer dependency in all other blocks.

I don't think it makes sense to add a `webpack` peer dependency to every block. And basically that peer dependency isn't correct, since the block does not require webpack. But maybe I didn't get your point.

> (...) if have any unpublished changes

Nope, should all be published. So we would then have an `extract-text` block v1 for webpack v2 and a v2 for webpack v3, right?
(It's not really intuitive, but aligning with the webpack major versions won't make sense here; if anything we might consider aligning it with the `extract-text-plugin` versions, but that doesn't seem very handy either)

**Comment by sapegin** [comment]
> I don't think it makes sense to add a webpack peer dependency to every block

I did say “add”, I said “update” in places where we have it ;-)

> So we would then have an extract-text block v1 for webpack v2 and a v2 for webpack v3, right?

👍

File: packages/extract-text/package.json [hunk 1]
```diff
@@ -19,15 +19,15 @@
   "repository": "andywer/webpack-blocks",
   "bugs": "https://github.com/andywer/webpack-blocks/issues",
   "dependencies": {
-    "extract-text-webpack-plugin": "^2.1.2"
+    "extract-text-webpack-plugin": "^3.0.0"
   },
   "peerDependencies": {
-    "webpack": ">= 2.0.0"
+    "webpack": ">= 3.0.0"
```
[the remark this answers is not in the document]
Comment (sapegin) [reply packages/extract-text/package.json]: Then we should remove it from every block that has it.

**Comment by andywer** [comment]
Added a commit to highlight webpack version support.

**Comment by sapegin** [comment]
@andywer Feels like a lot of text ;-) How about something [like this](https://github.com/webpack-contrib/extract-text-webpack-plugin#install) or a table?

**Comment by andywer** [comment]
@sapegin Changed it :)

**Comment by andywer** [comment]
@sapegin Can this be merged?

**Comment by sapegin** [comment]
Yeah, I think so.

**Pull request merged**